### PR TITLE
feat: xblock asides view for rendering in an iframe in MFE

### PR DESCRIPTION
**Pull request by Anas12091101**
## Description

This PR introduces an aside only studio view in edx-platform for any Xblock i.e, given an Xblock usage key, get all the asides and render them into an HTTP response template page. This can then be utilized into an iframe in the course authoring MFE which currently doesn't support any asides configuration.

Useful information to include:

- The change in this PR will impact the course author using course-authoring MFE.
- Screenshots:
Screenshot 2024-01-05 at 3 57 27 PM
Screenshot 2024-01-05 at 3 59 47 PM

## Supporting information

Related [Issue](https://github.com/openedx/frontend-app-course-authoring/issues/545)

## Testing instructions

Please provide detailed step-by-step instructions for testing this change.
- Login to edx-platform.
- In Studio go to the edX Demo Course. Create a new unit which is a multiple choice problem.
- Confgure rapid response aside for this problem from the steps mentioned in it's [repository](https://github.com/mitodl/rapid-response-xblock).
- Go to the URL http://localhost:18010/asides/{xblock-usage-key-for-the-problem} and you will see the rapid response aside upon loading completion.

### CODE REVIEW

File: cms/lib/xblock/tagging/tagging.py [hunk 1]
```diff
@@ -71,6 +71,48 @@ def student_view_aside(self, block, context):  # pylint: disable=unused-argument
             return fragment
         else:
             return Fragment('')
+
```
Comment (Anas12091101) [inline cms/lib/xblock/tagging/tagging.py:74]: Testing multiple studio_view asides

File: cms/lib/xblock/tagging/tagging.py [hunk 2]
```diff
@@ -71,6 +71,48 @@ def student_view_aside(self, block, context):  # pylint: disable=unused-argument
...
+                    'current_values': ["tag_current_values"],
+                })
+            fragment = Fragment(render_to_string('structured_tags_block.html', {'tags': tags,
+                                                                                'tags_count': len(tags),
+                                                                                'block_location': block.location}))
+            fragment.add_javascript_url(self._get_studio_resource_url('/js/xblock_asides/structured_tags.js'))
+            fragment.initialize_js('StructuredTagsInit')
+            return fragment
+        else:
+            return Fragment('')
```
Comment (Anas12091101) [inline cms/lib/xblock/tagging/tagging.py:115]: Testing multiple asides

**Comment by pdpinch** [comment]
@Anas12091101 can you clarify this? I believe this view was functional in the old Studio UI, but it doesn't work in the course authoring MFE. So this is restoring existing functionality in the new MFE UI.

**Comment by Anas12091101** [comment]
@pdpinch yes, you are right. In the legacy studio we can access this view in the plugins tab which displays all the available plugins for the xblock installed in the edx-platform. But in the new `course authoring MFE`, there is no method to set the values of the plugins. The approach used in this PR is inspired by the one used by the `learning MFE` which loads the `student_view` along with the `asides` in an iframe. However there's one change. We don't need to load the legacy `studio_view` of the problem xblock as the course authoring is already parsing the problem and rendering it into it's own components. Hence we only need the xblock's `asides studio_view` which can easily be rendered in an iframe in course authoring MFE.

**Pull request closed without merging**